src/_includes/docs/install/compiler/xcode.md

```diff
@@ -98,7 +92,26 @@ To [install and set up CocoaPods][cocoapods], run the following commands:
    {{prompt1}} brew install cocoapods
    ```
 
-Using Homebrew reduces potential issues with chipsets and install permissions.
+1. Add CocoaPods to your macOS `PATH` using the following command.
+
+    ```terminal
+    $ cat <<EOF >> ~/.zshenv
+    # Configure CocoaPods
+
+    export GEM_HOME=\$HOME/.gem
+    if [[ \$PATH != *"\$GEM_HOME"* ]]
+    then
+        export PATH=\$GEM_HOME/bin:\$PATH
+    fi
+    EOF
```

Comment:
We don't need to dictate how the user should add a directory to their path in their `.zshenv` file. I would prefer give instructions that align more with the [Cocoapods Getting Started guide](https://guides.cocoapods.org/using/getting-started.html) like this:

```
export GEM_HOME=$HOME/.gem
export PATH=$GEM_HOME/bin:$PATH
```

Reply:
The extra code around this was to keep the PATH sane if the user runs `source` more than once.

Reply:
I still think this is unnecessary code to make the user copy-paste. It's also not very readable. I know I spent a while staring at it before I understood what it was doing. But if you ad just said "Add this to your path in `.zshenv` I would have been able to do it with whichever editor I normally use, and add it in a place that makes sense, perhaps with a comment. We don't need to hand-hold the user through this.

Reply:
I changed this to opening a text editor and editing the `~/.zshenv` file.